src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AppointmentController.kt

```diff
@@ -89,5 +91,8 @@ class AppointmentController(private val appointmentService: AppointmentService)
       )
     ]
   )
-  fun deleteAppointment(@PathVariable(value = "id") id: Long) = appointmentService.deleteAppointment(id)
+  fun deleteAppointment(
+    @PathVariable(value = "id") id: Long,
+    @ApiParam(value = "Whether to delete the whole sequence of recurring appointments to which this appointment belongs.") @RequestParam(value = "deleteRelatedAppointments", required = false, defaultValue = "true") deleteRelatedAppointments: Boolean
+  ) = appointmentService.deleteAppointment(id, deleteRelatedAppointments)
```

Comment:
i think we ought to have a separate endpoint to delete all recurring appointments, as we could then use the recurring appointment id which would prevent any nasties around a single appointment being deleted. I was going to do that next

Reply:
Good shout